### PR TITLE
change display() to print() and Fix invalid syntax error in VAERSFile…

### PR DESCRIPTION
**Pull request by testtestingtester**
…Downloader.py

>>> from IPython.display import display
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named IPython.display

### CODE REVIEW

File: src/VAERSFileDownloader.py [hunk 1]
```diff
@@ -58,5 +58,5 @@ def downloadVAERSFileAndUnzip(file, workingDirectory):
 
 def updateVAERSFiles(years, workingDirectory):
     for year in years:
-        downloadVAERSFileAndUnzip(f'{year}VAERSData.zip', workingDirectory)
```
Comment (KnollFrank) [inline src/VAERSFileDownloader.py:61]: What is the problem with  f-Strings?

File: src/VAERSFileDownloader.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 def solveCaptchaAndStartFileDownload(driver, captchaImageFile):
     saveCaptchaImageAs(driver, captchaImageFile)
     textInCaptchaImage = getTextInCaptchaImage(captchaImageFile)
-    display('textInCaptchaImage: ', textInCaptchaImage)
+    print('textInCaptchaImage: ', textInCaptchaImage)
```
Comment (KnollFrank) [inline src/VAERSFileDownloader.py:19]: remove space after colon: print('textInCaptchaImage:', textInCaptchaImage)

File: src/VAERSFileDownloader.py (hunk 1 above)
Comment (testtestingtester) [reply src/VAERSFileDownloader.py:61]: Hey just saw your message, the problem was the regular python interpreter doesn't have that, iPython does though

File: src/VAERSFileDownloader.py (hunk 2 above)
Comment (testtestingtester) [reply src/VAERSFileDownloader.py:19]: okay sure that sounds good

**Comment by testtestingtester** [comment]
Thanks for all your great work on the project! Please check out this project I started few days ago it's already has 300 people using it right now :) https://fetvguide.com enjoy your day!

**Pull request closed without merging**